src/table/boot.rs

```diff
@@ -672,6 +681,42 @@ impl BootServices {
         unsafe { (self.set_watchdog_timer)(timeout, watchdog_code, data_len, data) }.into()
     }
 
+    /// A direct translation of the UEFI `ConnectController` method.
+    ///
+    /// Usually one disconnects and then reconnects certain drivers
+    /// to make them rescan some state that changed, e.g. reconnecting
+    /// a `BlockIO` handle after your app changed the partitions somehow.
+    pub fn connect_controller(
+        &self,
+        controller: Handle,
+        driver_image: Option<Handle>,
+        remaining_device_path: Option<&DevicePath>,
+        recursive: bool,
+    ) -> Result {
+        unsafe {
+            (self.connect_controller)(
+                controller,
+                driver_image,
+                remaining_device_path.map(|dp| dp as _).unwrap_or(null()),
```

Comment:
Very minor nit: the style in the rest of the file is to call it as `ptr::null()`.